_data/companies.csv

```diff
@@ -29,9 +30,12 @@ Diploid,http://www.diploid.com/,Bioinformatics,"Human genome interpretation"
 Bright Security,https://brightsec.com/,Cyber Security,"Machine Learning powered advanced protocol fuzzer"
 MESO,https://meso.design,Media Systems,"We design digital systems for communicative spaces"
 Play it My way,https://playitmyway.org/,Media Systems,"Amazon movies/youtube EDL manager to watch with objectionable scenes removed backend. Edit the profanities out of movies!"
+Crunchy Bridge,https://www.crunchybridge.com/,Communication & Networking,"The Developer Experience you always wanted for Postgres"
```

Comment:
The tagline doesn't really explain what this thing does apart from being something for Postgres..
```suggestion
Crunchy Bridge,https://www.crunchybridge.com/,Communication & Networking,"Fully managed PostgreSQL server in the cloud with a focus on DX"
```